package/base-files/files/lib/functions/ipv4.sh

```diff
@@ -0,0 +1,146 @@
+uint_max=4294967295
```

Comment:
Is there an advantage to include the shebang here? I know this is an included/sourced script, but would it ever be executed directly?

Reply:
It only includes function definitions, so nothing that would get executed because there’s no main body.